serializer.rst

```diff
@@ -1344,6 +1344,31 @@ normalizers (in order of priority):
     This denormalizer converts an array of arrays to an array of objects
     (with the given type). See :ref:`Handling Arrays <serializer-handling-arrays>`.
 
+    ByUsing the PropertyInfoExtractor you can simply annotate the arrays by '@var Person[]'
+
+    .. configuration-block::
+
+        .. code-block:: php-standalone
+
+            use Symfony\Component\PropertyInfo\Extractor\PhpDocExtractor;
+            use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+            use Symfony\Component\PropertyInfo\PropertyInfoExtractor;
+            use Symfony\Component\Serializer\Encoder\JsonEncoder;
+            use Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactory;
+            use Symfony\Component\Serializer\Mapping\Loader\AttributeLoader;
+            use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
+            use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+            use Symfony\Component\Serializer\Serializer;
+    
+            $reflectionExtractor = new ReflectionExtractor();
+            $phpDocExtractor = new PhpDocExtractor();
+            $propertyInfo = new PropertyInfoExtractor([], [$phpDocExtractor, $reflectionExtractor]);
+            
+            $classMetadataFactory = new ClassMetadataFactory(new AttributeLoader());
+            $normalizers = [new ObjectNormalizer($classMetadataFactory, null, null, $propertyInfo), new ArrayDenormalizer()];
+            
+            $this->serializer = new Serializer($normalizers, [new JsonEncoder()]);
```

Comment:
Given the goal of the example, I think we could provide a more compact snippet.

```suggestion
            $propertyInfo = new PropertyInfoExtractor([], [new PhpDocExtractor(), new ReflectionExtractor()]);
            $normalizers = [new ObjectNormalizer(new ClassMetadataFactory(new AttributeLoader()), null, null, $propertyInfo), new ArrayDenormalizer()];
            
            $this->serializer = new Serializer($normalizers, [new JsonEncoder()]);
```

Reply:
I did this change while merging. Thanks Javier.